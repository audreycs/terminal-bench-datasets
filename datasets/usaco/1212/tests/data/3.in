1000 2
58 77 153 134
1685 1754 1832 1763
1074 1037 1115 1153
412 363 442 488
1 1 4 4
230 232 310 307
206 255 333 284
1030 1083 1161 1108
235 227 305 312
1920 1845 1923 1998
1195 1232 1309 1273
56 79 155 132
543 558 636 620
1407 1344 1422 1485
256 206 283 333
208 253 331 286
923 866 944 1001
1694 1745 1823 1772
1235 1192 1271 1312
1873 1891 1970 1952
1359 1392 1469 1437
98 36 113 176
1352 1399 1477 1430
720 704 783 799
1226 1200 1278 1302
1579 1530 1606 1658
190 190 191 191
721 703 782 800
37 97 174 114
1722 1717 1795 1801
1347 1404 1482 1425
215 247 325 292
1040 1073 1152 1117
38 96 173 115
1202 1222 1302 1280
1240 1187 1266 1317
214 248 326 291
1038 1078 1173 1146
1037 1075 1154 1115
867 922 1000 945
374 403 481 451
927 862 940 1005
877 912 989 956
258 204 281 335
52 83 160 128
544 557 635 621
863 926 1004 941
1060 1052 1130 1138
1053 1059 1137 1131
1576 1533 1609 1654
869 920 997 947
850 850 851 851
389 389 468 464
1757 1682 1760 1835
538 564 643 615
1532 1576 1655 1610
1409 1342 1420 1487
241 221 298 318
710 714 792 789
1222 1204 1282 1298
1894 1870 1948 1973
410 366 444 485
882 908 985 960
75 60 136 151
1690 1749 1827 1768
377 400 478 454
376 401 479 453
416 359 437 494
1731 1708 1786 1810
1225 1201 1279 1301
537 565 644 614
514 587 665 592
1892 1872 1950 1971
1058 1054 1132 1136
1251 1176 1254 1328
530 571 650 607
687 737 815 767
70 65 141 146
860 929 1007 938
263 199 276 341
1355 1396 1474 1433
685 739 817 764
1885 1879 1957 1964
896 893 971 974
1244 1183 1261 1321
1558 1550 1628 1637
1216 1210 1289 1292
521 541 637 611
1861 1903 1982 1939
905 884 962 983
1021 1093 1170 1098
1026 1087 1165 1103
1754 1685 1764 1832
523 579 657 599
1072 1039 1118 1151
914 875 953 992
94 40 118 171
512 589 667 590
1237 1190 1269 1314
1734 1705 1783 1813
1031 1082 1160 1109
1730 1709 1787 1809
352 423 501 431
357 418 496 436
237 225 303 314
1583 1527 1603 1662
1901 1863 1941 1980
1563 1546 1624 1641
229 233 311 306
1890 1874 1952 1969
51 84 161 127
1406 1345 1423 1484
40 95 172 116
1081 1030 1108 1160
48 87 164 124
540 562 640 617
1706 1733 1811 1785
1568 1541 1619 1646
558 543 621 635
1592 1517 1595 1670
421 354 432 499
220 242 320 297
1549 1560 1638 1628
698 726 805 777
1402 1349 1427 1480
1593 1516 1594 1671
394 381 463 469
1587 1523 1599 1666
194 267 345 272
1883 1881 1959 1962
711 713 791 790
60 75 151 136
213 249 327 290
1059 1053 1131 1137
1708 1730 1809 1787
1574 1592 1618 1660
249 213 290 326
857 932 1010 935
1686 1753 1831 1764
41 94 171 117
724 700 768 784
419 356 434 497
1858 1906 1985 1936
575 525 602 654
1231 1196 1274 1308
884 905 983 962
1087 1025 1103 1165
224 238 316 301
424 351 429 502
1905 1859 1937 1984
1751 1687 1766 1830
1083 1029 1107 1161
1903 1861 1939 1982
1561 1547 1625 1640
1020 1088 1146 1127
1176 1251 1328 1254
670 670 829 829
1362 1389 1466 1440
239 223 301 316
1585 1525 1601 1664
1082 1042 1116 1166
865 924 1002 943
1408 1343 1421 1486
29 104 182 107
1185 1241 1319 1264
238 224 302 315
929 860 938 1007
1357 1394 1472 1435
579 521 598 658
1033 1080 1158 1111
398 377 459 473
1755 1684 1763 1833
1914 1850 1928 1993
1364 1387 1464 1442
1723 1716 1794 1802
1205 1220 1299 1282
880 907 999 954
1175 1252 1330 1253
1682 1757 1835 1760
18 18 19 19
1725 1714 1792 1804
1839 1839 1842 1842
706 718 796 786
2 2 3 3
198 263 341 276
1710 1731 1762 1796
415 360 438 493
1856 1908 1987 1934
1919 1846 1924 1997
1247 1180 1258 1324
915 874 952 993
1179 1248 1325 1257
384 393 472 460
247 215 292 324
1496 1496 1501 1501
1414 1337 1415 1492
1513 1513 1514 1514
846 846 847 847
1410 1341 1419 1488
391 387 466 466
1043 1070 1149 1120
1389 1361 1439 1468
1494 1494 1495 1495
39 48 116 157
1750 1688 1767 1829
1889 1875 1953 1968
1867 1897 1976 1946
1413 1338 1416 1491
1535 1573 1651 1614
903 886 964 981
1517 1590 1669 1595
535 567 646 612
532 569 648 609
102 32 109 180
1340 1411 1489 1418
1564 1545 1623 1642
1726 1713 1791 1805
1207 1219 1298 1283
1354 1397 1475 1432
913 876 954 991
1250 1177 1255 1327
422 353 431 500
1403 1348 1426 1481
1077 1034 1112 1156
1029 1084 1162 1107
1028 1085 1163 1105
541 561 639 618
1227 1199 1277 1304
738 684 764 816
1864 1900 1979 1942
587 513 591 665
911 878 956 989
1078 1033 1111 1157
68 67 143 144
1212 1214 1293 1288
1371 1379 1457 1449
1716 1723 1802 1794
1232 1228 1284 1261
1370 1380 1458 1448
1387 1363 1441 1466
246 216 293 323
840 840 841 841
1909 1855 1933 1988
200 261 339 278
1733 1706 1784 1812
858 931 1009 936
584 516 594 662
886 903 981 964
222 240 318 299
917 872 950 995
250 212 289 327
737 685 765 815
581 518 596 660
746 677 756 824
1536 1572 1650 1615
1846 1918 1997 1924
1221 1205 1283 1297
1866 1898 1977 1945
259 203 280 336
1898 1866 1944 1977
242 220 297 319
355 420 498 434
228 234 312 305
1177 1250 1327 1255
373 404 482 450
370 407 485 447
1339 1412 1490 1417
1201 1223 1303 1279
100 34 111 178
212 250 328 289
1396 1354 1432 1475
1554 1555 1633 1633
1245 1182 1260 1322
255 207 284 332
551 550 628 628
1499 1499 1500 1500
1526 1582 1661 1604
418 357 435 496
1518 1589 1668 1596
852 852 853 853
1089 1023 1101 1168
516 585 662 594
1569 1540 1616 1647
93 41 119 170
360 416 494 438
513 588 666 591
582 517 595 661
1560 1548 1626 1639
261 201 278 338
82 53 130 158
1188 1239 1317 1266
902 887 965 980
88 46 124 165
1509 1509 1510 1510
673 750 828 751
555 546 624 632
700 724 802 779
16 16 1673 1673
838 838 839 839
1368 1383 1460 1446
1229 1197 1275 1306
1912 1852 1930 1991
1729 1710 1788 1808
1036 1076 1155 1114
1543 1566 1644 1621
252 210 287 329
1900 1864 1942 1979
388 390 469 463
529 572 651 606
726 697 777 804
1214 1212 1291 1290
1520 1587 1666 1598
1013 1013 1332 1332
1080 1031 1109 1159
1390 1360 1438 1469
1248 1179 1257 1325
556 545 623 633
1242 1185 1264 1319
1388 1362 1440 1467
569 531 608 647
63 72 148 139
1183 1243 1321 1262
1219 1207 1286 1295
1699 1740 1818 1777
1384 1366 1444 1462
1849 1915 1994 1927
1391 1359 1437 1470
1546 1563 1641 1624
86 49 126 163
526 576 654 602
226 236 314 303
1025 1089 1166 1102
536 566 645 613
64 71 147 140
1875 1889 1968 1954
359 417 495 437
233 229 307 310
892 897 975 970
562 538 617 639
1721 1718 1796 1800
1523 1585 1664 1601
894 895 973 972
1566 1543 1621 1644
45 90 167 121
1032 1081 1159 1110
1753 1692 1797 1778
683 741 819 762
379 398 477 455
1385 1365 1443 1464
1084 1028 1106 1162
1196 1231 1308 1274
1689 1750 1828 1767
1027 1086 1164 1104
1918 1847 1925 1996
1746 1693 1771 1825
393 384 464 468
1886 1878 1956 1965
211 251 329 288
1400 1351 1429 1478
879 910 987 958
909 880 958 987
1360 1391 1468 1438
585 515 593 663
1239 1188 1267 1316
1049 1063 1141 1126
1217 1209 1288 1293
403 372 453 479
369 408 486 446
1210 1216 1295 1286
1215 1211 1290 1291
571 529 606 649
1872 1892 1971 1951
692 733 811 771
712 712 790 791
518 575 664 653
873 916 993 951
1556 1553 1631 1635
559 542 620 636
404 371 452 480
546 555 633 623
1557 1551 1629 1636
34 100 178 111
367 410 488 444
550 551 629 627
718 705 784 798
1530 1578 1657 1608
205 256 334 283
568 532 609 645
1860 1904 1983 1938
699 725 804 778
1683 1756 1834 1761
202 259 337 280
351 424 502 430
1194 1233 1310 1272
1541 1567 1645 1620
735 688 767 813
74 61 137 150
916 873 951 994
1228 1198 1276 1305
11 11 12 12
349 426 504 427
1411 1340 1418 1489
1855 1909 1988 1933
1519 1588 1667 1597
1853 1911 1990 1931
25 25 26 26
1064 1048 1126 1142
92 42 120 169
1880 1884 1962 1959
533 580 613 646
265 197 274 343
1211 1215 1294 1287
697 727 806 776
1565 1544 1622 1643
918 871 949 996
1897 1867 1945 1976
387 391 470 462
1871 1893 1972 1950
716 707 786 795
417 358 436 495
691 734 812 770
76 59 135 152
201 260 338 279
681 743 821 760
1204 1230 1329 1303
1870 1894 1973 1949
900 889 967 978
1869 1895 1974 1948
1397 1353 1431 1476
1511 1511 1672 1672
1386 1364 1442 1465
1845 1919 1998 1923
861 928 1006 939
1582 1528 1604 1661
731 692 772 809
1213 1213 1292 1289
267 194 272 345
1377 1373 1451 1455
895 894 972 973
1580 1552 1630 1656
23 23 184 184
248 214 291 325
1736 1703 1781 1815
872 917 994 950
1844 1921 1999 1922
1674 1674 1675 1675
1681 1758 1836 1759
1906 1858 1936 1985
254 208 285 331
1577 1532 1608 1655
1567 1542 1620 1645
669 669 830 830
1525 1583 1662 1603
1401 1350 1428 1479
1369 1382 1459 1447
1363 1388 1465 1441
1042 1071 1150 1119
1342 1409 1487 1420
750 673 751 828
1705 1734 1812 1784
552 549 627 629
1218 1208 1287 1294
868 921 998 946
1745 1694 1772 1824
1756 1683 1761 1834
1840 1840 1841 1841
1742 1697 1775 1821
187 187 188 188
694 731 809 773
510 510 511 511
1527 1581 1660 1605
1584 1526 1602 1663
350 425 503 429
1057 1055 1133 1135
1921 1844 1922 1999
1521 1534 1611 1600
545 556 634 622
1743 1696 1774 1822
1552 1557 1635 1631
722 702 781 801
390 388 467 465
1899 1865 1943 1978
1879 1885 1964 1958
1404 1347 1425 1482
732 691 771 810
1361 1390 1467 1439
1717 1722 1801 1795
1540 1568 1646 1619
1334 1334 1335 1335
1238 1189 1268 1315
1398 1381 1471 1463
1878 1886 1965 1957
55 80 156 131
365 364 451 486
1365 1386 1463 1443
1190 1237 1315 1268
1208 1218 1297 1284
8 8 9 9
1506 1506 1507 1507
1693 1746 1824 1771
1187 1240 1318 1265
411 365 443 487
1916 1920 1963 1944
46 89 166 122
97 37 114 175
1758 1681 1759 1836
1220 1206 1285 1296
1559 1549 1627 1638
1698 1741 1819 1776
209 252 330 287
1382 1368 1446 1460
1353 1398 1476 1431
1852 1912 1991 1930
426 349 427 504
1393 1357 1435 1472
1704 1735 1813 1783
1034 1079 1157 1112
1186 1245 1301 1311
1412 1339 1417 1490
557 544 622 634
676 698 778 754
702 722 800 781
1044 1069 1147 1121
730 693 773 808
1063 1049 1127 1141
1346 1405 1483 1424
1073 1038 1117 1152
733 690 770 811
1399 1352 1430 1477
891 898 976 969
1589 1521 1598 1667
54 81 157 130
227 235 313 304
22 22 505 505
588 512 590 667
1512 1512 1515 1515
420 355 433 498
1843 1843 2000 2000
196 265 343 274
1199 1225 1305 1277
734 689 769 812
893 896 974 971
1067 1045 1123 1145
522 581 658 598
30 103 181 108
1086 1026 1104 1164
682 742 820 761
53 82 159 129
32 58 177 159
71 64 140 147
715 708 787 794
1679 1679 1838 1838
1014 1014 1331 1331
90 44 122 167
1578 1531 1607 1657
392 385 465 467
1739 1700 1778 1818
1907 1857 1935 1986
1744 1695 1773 1823
859 930 1008 937
566 534 612 643
1688 1751 1829 1766
1233 1194 1273 1309
1209 1217 1296 1285
1874 1890 1969 1953
189 189 348 348
1046 1067 1144 1123
1344 1407 1485 1422
1180 1247 1324 1258
1741 1698 1776 1820
864 925 1003 942
1863 1901 1980 1941
385 383 456 477
83 52 129 160
89 45 123 166
77 57 134 153
708 715 794 788
928 861 939 1006
574 526 603 652
361 415 493 439
713 711 789 792
371 406 484 448
1246 1181 1259 1323
1590 1520 1597 1668
1056 1056 1134 1134
1533 1575 1654 1612
1553 1556 1634 1632
1189 1238 1316 1267
1904 1860 1938 1983
1550 1559 1637 1629
240 222 299 317
881 909 986 959
871 918 995 949
104 29 107 182
103 30 108 181
1738 1701 1779 1817
1338 1413 1491 1416
380 397 476 456
564 536 615 641
739 683 762 817
66 69 145 142
1547 1562 1640 1625
1720 1719 1798 1799
223 239 317 300
1358 1393 1470 1436
1395 1355 1433 1474
216 246 324 293
925 864 942 1003
709 716 763 818
743 679 758 822
362 414 492 440
79 31 158 173
1192 1235 1312 1270
219 243 321 296
1076 1035 1113 1155
1862 1902 1981 1940
527 574 653 603
897 892 970 975
207 254 332 285
1405 1346 1424 1483
539 563 642 616
408 368 446 483
1687 1752 1830 1765
402 373 454 478
210 196 300 340
1069 1043 1121 1148
253 209 286 330
736 686 766 814
1230 1229 1313 1329
5 5 1678 1678
264 198 275 342
1070 1041 1120 1149
397 378 460 472
554 547 625 631
221 241 319 298
887 902 980 965
1545 1564 1642 1623
932 857 935 1010
1374 1376 1454 1452
405 386 450 428
531 570 649 608
423 352 430 501
101 33 110 179
7 7 14 14
1019 1094 1171 1097
1184 1242 1320 1263
1054 1058 1136 1132
1380 1370 1448 1458
401 374 455 476
1061 1051 1129 1139
1051 1061 1139 1129
898 891 969 976
1538 1570 1648 1617
1200 1224 1304 1278
930 859 937 1008
1381 1369 1447 1459
1039 1074 1153 1116
217 245 323 294
1539 1569 1647 1618
572 528 605 650
1884 1880 1958 1963
1562 1522 1671 1648
549 552 630 626
1505 1505 1508 1508
1713 1726 1805 1791
1544 1565 1643 1622
1850 1914 1993 1928
908 881 959 986
1017 1017 1174 1174
1093 1019 1097 1172
203 258 336 281
1712 1727 1806 1790
1732 1707 1785 1811
1193 1234 1311 1271
193 268 346 271
920 869 947 998
1224 1202 1280 1300
1223 1203 1281 1299
1337 1414 1492 1415
67 68 144 143
1341 1410 1488 1419
1090 1022 1100 1169
717 706 785 797
269 192 270 347
1349 1402 1480 1427
745 678 757 823
845 845 1012 1012
354 421 499 433
1022 1092 1169 1099
199 262 340 277
185 185 186 186
1041 1072 1151 1118
1676 1676 1677 1677
1848 1916 1995 1926
1572 1537 1613 1651
358 399 448 491
1896 1868 1946 1975
926 863 941 1004
1719 1720 1799 1798
690 687 752 752
1088 1024 1102 1167
218 244 322 295
1692 1747 1825 1770
1859 1905 1984 1937
96 38 115 174
686 738 816 766
1343 1408 1486 1421
744 710 793 765
1737 1702 1780 1816
1249 1178 1256 1326
62 73 149 138
244 218 295 321
1383 1367 1445 1461
1531 1577 1656 1609
1696 1743 1821 1774
399 376 458 474
1085 1027 1105 1163
1747 1691 1770 1826
714 709 788 793
719 729 803 796
586 514 592 664
674 749 827 753
31 102 180 109
561 539 618 638
28 105 183 106
832 832 833 833
1197 1227 1307 1275
85 50 127 162
1524 1584 1663 1602
1877 1887 1966 1956
565 535 614 642
1847 1917 1996 1925
921 868 946 999
414 361 439 490
1748 1690 1769 1827
748 675 754 826
576 524 601 655
1068 1044 1122 1147
1700 1739 1817 1779
266 195 273 344
1502 1502 1503 1503
73 62 138 149
1571 1538 1614 1650
1047 1066 1143 1124
509 509 668 668
1182 1244 1322 1260
1079 1032 1110 1158
573 527 604 651
381 396 475 457
1075 1036 1114 1154
1529 1579 1658 1607
1711 1728 1807 1789
933 856 934 1011
583 520 641 666
42 93 170 118
547 554 632 624
1709 1729 1808 1788
1888 1876 1954 1967
50 85 162 126
87 47 125 164
262 200 277 339
1913 1851 1929 1992
1714 1725 1804 1792
922 867 945 1000
1045 1068 1145 1122
1522 1586 1665 1599
1752 1686 1765 1831
1092 1020 1098 1171
1394 1356 1434 1473
35 99 176 112
1857 1907 1986 1935
912 877 955 990
705 719 797 785
1702 1737 1815 1781
1178 1249 1326 1256
204 257 335 282
368 409 487 445
236 226 304 313
81 54 131 156
425 350 428 503
904 885 963 982
906 883 961 984
1372 1378 1456 1450
848 848 855 855
1581 1529 1605 1659
1241 1186 1265 1318
1516 1593 1670 1594
1588 1591 1617 1627
1735 1704 1782 1814
723 701 780 802
560 540 619 637
878 911 988 957
65 70 146 141
1895 1869 1947 1974
1350 1401 1479 1428
519 583 660 596
1917 1848 1926 1995
1252 1175 1253 1330
1703 1736 1814 1782
1048 1065 1142 1125
232 230 308 309
47 88 165 123
1052 1060 1138 1130
1024 1090 1167 1101
520 582 659 597
1893 1871 1949 1972
577 523 600 656
1695 1744 1822 1773
689 735 813 769
548 553 631 625
1910 1854 1932 1989
195 266 344 273
1376 1374 1452 1454
78 56 133 154
931 858 936 1009
693 732 810 772
741 681 760 820
1392 1358 1436 1471
508 508 843 843
1697 1742 1820 1775
534 568 647 610
701 723 801 780
36 98 175 113
1854 1910 1989 1932
1065 1047 1125 1143
234 228 306 311
1724 1715 1793 1803
883 906 984 961
885 904 982 963
80 55 132 155
382 395 474 458
1035 1077 1156 1113
99 35 112 177
400 375 457 475
1018 1095 1172 1096
679 745 823 758
386 392 471 461
688 736 814 768
1749 1689 1768 1828
44 91 168 120
567 533 611 644
696 728 807 775
17 17 20 20
1718 1721 1800 1797
61 74 150 137
1023 1091 1168 1100
1715 1724 1803 1793
524 578 656 600
1908 1856 1934 1987
570 530 607 648
375 402 480 452
837 837 842 842
678 746 824 757
862 927 1005 940
21 21 506 506
834 834 835 835
260 202 279 337
363 413 491 441
1691 1748 1826 1769
507 507 1504 1504
1497 1497 1498 1498
528 573 652 604
69 66 142 145
542 559 638 619
1570 1539 1615 1649
749 674 753 827
844 844 1493 1493
243 219 296 320
677 747 825 756
409 367 445 484
907 882 960 985
1573 1536 1612 1652
589 560 610 605
856 933 1011 934
849 849 854 854
1191 1236 1314 1269
866 923 1001 944
831 831 836 836
43 92 169 119
517 584 661 595
1882 1882 1960 1961
1727 1712 1790 1806
1055 1057 1135 1133
1868 1896 1975 1947
874 915 992 952
1375 1375 1453 1453
1851 1913 1992 1929
1865 1899 1978 1943
515 586 663 593
728 695 775 806
563 537 616 640
1591 1519 1596 1669
1095 1064 1148 1106
889 900 978 967
1091 1021 1099 1170
1234 1193 1272 1310
1348 1403 1481 1426
24 24 27 27
1701 1738 1816 1780
1373 1377 1455 1451
395 380 462 470
268 193 271 346
671 671 672 672
1379 1371 1449 1457
553 548 626 630
704 720 798 783
1378 1372 1450 1456
1367 1384 1461 1445
245 217 294 322
1066 1046 1124 1144
197 264 342 275
57 78 154 133
888 901 979 966
49 86 163 125
1534 1574 1653 1613
740 682 761 819
225 237 315 302
1094 1018 1096 1173
1333 1333 1336 1336
1548 1561 1639 1626
1891 1873 1951 1970
525 577 655 601
680 744 822 759
1680 1680 1837 1837
366 411 489 443
1537 1571 1649 1616
1351 1400 1478 1429
1915 1849 1927 1994
1015 1015 1016 1016
725 699 779 803
356 419 497 435
407 369 447 482
413 362 441 489
578 522 599 657
231 231 309 308
910 879 957 988
257 205 282 334
707 717 795 787
1551 1558 1636 1630
684 740 818 763
383 394 473 459
353 422 500 432
1586 1524 1600 1665
1203 1221 1300 1281
924 865 943 1002
1555 1554 1632 1634
1902 1862 1940 1981
1740 1699 1777 1819
1366 1385 1462 1444
1881 1883 1961 1960
59 76 152 135
72 63 139 148
1236 1191 1270 1313
192 269 347 270
1887 1877 1955 1966
95 39 117 172
727 696 776 805
901 888 966 979
1911 1853 1931 1990
695 730 808 774
1345 1406 1484 1423
1542 1518 1652 1611
84 51 128 161
703 721 799 782
580 519 597 659
1206 1195 1263 1307
378 382 440 492
747 676 755 825
1071 1040 1119 1150
1198 1226 1306 1276
1181 1246 1323 1259
875 914 991 953
10 10 13 13
742 680 759 821
1528 1580 1659 1606
6 6 15 15
1707 1732 1810 1786
919 870 948 997
729 694 774 807
1575 1535 1610 1653
364 412 490 442
1876 1888 1967 1955
1684 1755 1833 1762
870 919 996 948
1062 1050 1128 1140
396 379 461 471
675 748 826 755
33 101 179 110
1356 1395 1473 1434
91 43 121 168
1728 1711 1789 1807
876 913 990 955
372 405 483 449
1243 1184 1262 1320
899 890 968 977
890 899 977 968
1050 1062 1140 1128
251 211 288 328
406 370 449 481
105 28 106 183
