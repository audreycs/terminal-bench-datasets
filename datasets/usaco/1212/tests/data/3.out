70036 70011
